test/common/buffer/buffer_fuzz.cc

```diff
@@ -119,7 +119,8 @@ class StringBuffer : public Buffer::Instance {
     uint64_t size_copied = 0;
     uint64_t num_slices_copied = 0;
     while (size_copied < length && num_slices_copied < num_slices) {
-      auto copy_length = std::min((length - size_copied), slices[num_slices_copied].len_);
+      auto copy_length = std::min((length - size_copied),
+                                  static_cast<unsigned long long>(slices[num_slices_copied].len_));
```

Comment:
s/unsigned long long/uint64_t/

I see why this might be needed: `Buffer::RawSlice::len_` is declared as a `size_t`. One option is to switch that to a uint64_t, but try doing the above substitution and see if CI passes.